CHANGES_UNRELEASED.md

```diff
@@ -9,9 +9,19 @@
 
 ### What's New
 
-- `FirefoxAccount` is now deprecated
-- Introducing `FxAccountManager` which provides a higher-level interface to Firefox Accounts. Among other things, this class handles (and can recover from) authentication errors, exposes device-related account methods, handles its own keychain storage and fires observer notifications for important account events.
+- iOS: `FirefoxAccount` is now deprecated
+- iOS: Introducing `FxAccountManager` which provides a higher-level interface to Firefox Accounts. Among other things, this class handles (and can recover from) authentication errors, exposes device-related account methods, handles its own keychain storage and fires observer notifications for important account events.
+- Android: `FirefoxAccount.handlePushMessage` now handles all possible FxA push payloads and will return new `AccountEvent`s:
+  - `.ProfileUpdated` which should be handled by fetching the newest profile.
+  - `.AccountAuthStateChanged` should be handled by checking if the authentication state is still valid.
+  - `.AccountDestroyed` should be handled by removing the account information (no need to call `FirefoxAccount.disconnect`) from the device.
+  - `.DeviceConnected` can be handled by showing a "<Device name> is connected to this account" notification.
+  - `.DeviceDisconnected` should be handled by showing a "re-auth" state to the user if `isLocalDevice` is true. There is no need to call `FirefoxAccount.disconnect` as it will fail.
+
 
 ### Breaking changes
 
-- `FirefoxAccount.fromJSON(json: String)` has been replaced by the `FirefoxAccount(fromJsonState: String)` constructor.
+- iOS: `FirefoxAccount.fromJSON(json: String)` has been replaced by the `FirefoxAccount(fromJsonState: String)` constructor.
+- Android: A few changes were made in order to decouple device commands from "account events":
+  - `AccountEvent` enum has been refactored: `.TabReceived` has been replaced by `.IncomingDeviceCommand(IncomingDeviceCommand)`, `IncomingDeviceCommand` itself is another enum that contains `TabReceived`.
+  - `FirefoxAccount.pollDeviceCommands` now returns an array of `IncomingDeviceCommand`.
```

Comment:
While we're updating the changelog, could you please also add the `[#issuenumber]` link on the end of these items?